sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py

```diff
@@ -67,3 +72,17 @@ def test_product_packaging_qty_wo_packaging(self):
         self.assertEqual(order_line.product_packaging_qty, 0.0)
         with self.assertRaises(UserError):
             order_line.write({"product_packaging_qty": 3.0})
+
+    def test_product_packaging_qty_from_external(self):
```

Comment:
I am not sure to understand what more is tested with this function, than the beginning of the previous one ?

Reply:
Thanks for the comment. The previous ones have `product_uom_qty` of 3, which is less than and not divisible by packaging qty of 5. This test is to increase coverage for the case that `product_uom_qty` of 15 is divisible by 5.

Reply:
Ok thanks for the explanation